app/views/providers/income_summary/index.html.erb

```diff
@@ -23,16 +23,32 @@
             error: errors[transaction_type.name.to_sym]
           ) %>
     <% end %>
+
       <% if @legal_aid_application.transaction_types.credits.count < TransactionType.credits.count %>
         <%= render partial: 'add_other_income' %>
       <% end %>
+  </ol>
+
```

Comment:
Is there a `<hr>` missing here like:

![image](https://user-images.githubusercontent.com/25043924/104199272-2667fa80-541f-11eb-98d4-e7d37849974e.png)

Other Income...